README.md

```diff
@@ -6,9 +6,41 @@ This repository contains a couple of demos for core debugging stories, that need
 
 ## Usage
 
-Clone this repository, and run
+### TypeScript Projects
+
+The following projects have been adapted from [todomvc.com](https://todomvc.com/).
 
+#### TypeScript, React, Parcel
+
+```sh
+cd typescript-react
+yarn install
```

Comment:
Let's stick to either `npm` or `yarn`. I don't really feel strongly here, and so far always went with `npm` since it's readily available with every Node installation. Is there a good reason to prefer `yarn`?

Reply:
I don't feel strongly about either. I like yarn because of `yarn command` vs. `npm run command`. But I'm happy to switch to npm.

Reply:
Let's stick to `npm` then. We only need `npm install` and `npm start`, so no need to write `run` at all :-)

Reply:
Can you make the change to `npm` on this PR?